docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -14,7 +14,50 @@ This guide covers:
 
 This guide takes as input the application developed in the link:getting-started[Getting Started Guide].
 
-== Prerequisites
+== GraalVM
+
+Building a native executable requires using a distribution of GraalVM.
+There are two distributions: Oracle GraalVM and Mandrel.
+The differences between the two distributions are as follows:
+
+* Mandrel is a downstream distribution of the Oracle GraalVM community edition.
+Mandrel's main goal is to provide a way to build native executables specifically designed to support Quarkus.
+
+* Mandrel releases are built from a code base derived from the upstream Oracle GraalVM code base,
+with only minor changes but some significant exclusions.
+They support the same capabilities to build native executables as Oracle GraalVM,
+with no significant changes to functionality.
+Notably, they do not include support for polyglot programming.
+
+* Mandrel is built slightly differently to GraalVM, using the standard OpenJDK project.
+This means that it does not profit from a few small enhancements that Oracle have added to the version of OpenJDK used to build their own GraalVM downloads.
+
+* Mandrel is only available for projects that can be compiled with Java 11.
+If you are using Java 8,
+you should consider using Oracle GraalVM instead.
+
+* Mandrel is currently only recommended for containerized environments.
```

Comment:
Hello @galderz, is this sentence implying that the `native-image` *generation* needs to happen within a container or is this saying that the built native image needs to be run within a container?
Either way, I'm curious about the requirement of a containerized environment. In the context of Mandrel, is there some specific advantages that containerized environments provide? Are these advantages the same as probably Graal VM too provides in containerized environment?

Reply:
It's not required. But it's convenient. If you don't want containerized, you can just install a mandrel release from here and run with it (by pointing GRAALVM_HOME to the install): https://github.com/graalvm/mandrel/releases

We say that, because on Mac and Windows you can use the container to produce linux native images and deploy on Linux as no Windows and Mac community builds exist.

We only provide Linux builds so far. If somebody steps in to do Mac and Windows community builds even better. It probably won't be us doing those (in the foreseeable future). Note that for Mac builds one would have to do OpenJDK 11u vanilla mac builds too or use some other base JDK which would open up the Mac build to a different class of problems.

Reply:
@jaikiran I'm expanding the points here to make it clearer.

Reply:
The updated section looks much more clearer to me. Thank you @galderz